src/app/home/pages/PlansPage/components/Wizard/NamespacesTable.tsx

```diff
@@ -97,9 +97,25 @@ const NamespacesTable: React.FunctionComponent = () => {
     const editedNamespace = values.editedNamespaces.find(
       (editedNS) => editedNS.id === namespace.id
     );
+    const { currentPlan } = useSelector((state: DefaultRootState) => state.plan);
+    let isTargetNamespaceNameSet = false
+    let setTargetNamespaceName = ""
```

Comment:
Can you use a non-verb name? Seeing references to `setTargetNamespaceName` may seem at first like it's a function. Maybe `targetNamespaceFromPlan` or something